ubuntu/latest.Dockerfile

```
@@ -7,13 +7,151 @@ LABEL org.opencontainers.image.description DESCRIPTION="Ubuntu latest"
 LABEL org.opencontainers.image.source=https://github.com/lecaoquochung/docker-images/blob/main/ubuntu/latest.Dockerfile
 LABEL org.opencontainers.image.licenses=MIT
 
-# Update package index and install dependencies
+# Set environment variables for non-interactive installations
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Update package index and install basic dependencies
 RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)